dom-event-delegation/main.js

```diff
@@ -0,0 +1,16 @@
+var taskList = document.querySelector('.task-list');
+
+function clickFunction(event) {
+  console.log(event.target);
+  console.log(event.target.tagName);
+
+  var button = document.querySelector('button');
```

Comment:
This will only get the first button in the document. If you try this with any other entry but the one on top of the list, this won't work.

Reply:
![Kapture 2022-03-31 at 14 47 43](https://user-images.githubusercontent.com/99709530/161154883-1453b1f6-257a-48c9-8e2f-019bfe7e4eba.gif)


I got the result I wanted, but I don't think I got it the way I was supposed to via comparing the event.target. I'm wondering how to select the button element in the list